components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -33,12 +33,71 @@ module crm_physics
    public :: m2005_effradius
 
    integer :: crm_qaerwat_idx, crm_dgnumwet_idx
-   integer :: prec_dp_idx, snow_dp_idx, prec_sh_idx, snow_sh_idx
+   ! integer :: prec_dp_idx, snow_dp_idx, prec_sh_idx, snow_sh_idx
    integer :: prec_sed_idx, snow_sed_idx, snow_str_idx, prec_pcw_idx, snow_pcw_idx
-   integer :: cldo_idx
-#ifdef MAML
-   integer :: crm_pcp_idx,crm_snw_idx
-#endif
+   integer :: cldo_idx, cld_idx, concld_idx, iciwp_idx, iclwp_idx
+
+   ! Physics buffer fields normally registered by offline parameterizations
+   integer :: qme_idx
```

Comment:
Why do some `_idx` variables get initialized to `-1` (which makes sense because that's guaranteed to throw and error if used uninitialized) while these are not? Would prefer to see a consistent treatment here.

Reply:
The reason I did this was because I was moving variables over from other non-MMF files and I figured it might be best to keep the format that they were originally written with rather than adapt them to the format of the current file... but now I realize that was dumb, so I'll fix it.